controls/srg_gpos/SRG-OS-000163-GPOS-00072.yml

```diff
@@ -5,10 +5,16 @@ controls:
         title: '{{{ full_name }}} must terminate all network connections associated with
     a communications session at the end of the session, or as follows: for in-band
     management sessions (privileged sessions), the session must be terminated after
-    10 minutes of inacti'
+    10 minutes of inactivity; and for user sessions (non-privileged session),
+    the session must be terminated after 15 minutes of inactivity,
+    except to fulfill documented and validated mission requirements.'
         rules:
-            - sshd_set_idle_timeout
-            - sshd_set_keepalive
-            - sshd_set_keepalive_0
+            #- sshd_set_idle_timeout
+            #- sshd_set_keepalive
+            #- sshd_set_keepalive_0
             - accounts_tmout
-        status: automated
+            - var_accounts_tmout=10_min
+        # We don't have a way to configure ssh timeout on RHEL 9, the
+        # sshd_config options ClientAliveInterval and ClientAliveCountMax
+        # dont't work on RHEL 9 as they used to in RHEL 8.
+        status: partial
```

Comment:
Partial will not be valid for DISA. I think "does not meet" might be best for the control unfortunately.